test/lightning/invocation_test.exs

```diff
@@ -515,16 +520,15 @@ defmodule Lightning.InvocationTest do
     end
 
     test "list_work_orders_for_project/3 returns paginated workorders" do
-      project = project_fixture()
+      project = insert(:project)
       now = Timex.now()
 
-      fixture =
-        workflow_job_fixture(workflow_name: "chw-help", project_id: project.id)
-
-      job1 = fixture.job
+      workflow = insert(:workflow, name: "chw-help", project: project)
+      job1 = insert(:job, workflow: workflow)
+      trigger = insert(:trigger, workflow: workflow)
```

Comment:
As I mentioned above, you can use the `with_job|trigger` helpers to associate the job/trigger with a workflow and have the `insert()` persist the jobs/triggers when the workflow is created.